rs/ic_os/setupos_tool/src/main.rs

```diff
@@ -56,25 +57,19 @@ pub fn main() -> Result<()> {
 
     match opts.command {
         Some(Commands::GenerateNetworkConfig { output_directory }) => {
+            let setup_config: config::types::SetupOSConfig =
+                deserialize_config(DEFAULT_SETUPOS_CONFIG_OBJECT_PATH)?;
+
             let config_map = config_map_from_path(Path::new(&opts.config))
                 .context("Please specify a valid config file with '--config'")?;
             eprintln!("Using config: {:?}", config_map);
 
             let network_info = NetworkInfo::from_config_map(&config_map)?;
             eprintln!("Network info config: {:?}", &network_info);
```

Comment:
setupos_tool is still reading directly from config.ini for it's NetworkInfo struct. This is the last SetupOS change that needs to be made, and I'll do it after https://github.com/dfinity/ic/pull/1365/files merges